### DISCUSSION TITLE
What is your favorite food?

### DISCUSSION THREAD

**Discussion by yuhi25** [NAIST-SE/AttractingContributors#1151] [2024 Attraction Contributions]
https://github.com/yuhi25/food

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113

**Comment by ryosuke-sota** [comment]
I made a PR on your repo. </br>Could you contribute to mine? </br>Here : [my project link (fav drinks)](https://github.com/ryosuke-sota/AttractingContributors.git)  </br>Thank you.

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI.git
Thank you very much！

**Comment by kohanaist** [comment]
I have made a PR on your repo. 
Could you contribute to mine? 
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by r-nishioka** [comment]
I have made a PR on your repo. 
Could you contribute to mine? 
[Where would you like to trip in Japan?](https://github.com/r-nishioka/AttractingContributors-Trip.git)
